services/inventory/agents.go

```diff
@@ -704,6 +706,52 @@ func (as *AgentsService) ChangeRDSExporter(ctx context.Context, req *inventorypb
 	return res, nil
 }
 
+// AddExternalExporter inserts external-exporter Agent with given parameters.
+func (as *AgentsService) AddExternalExporter(req *inventorypb.AddExternalExporterRequest) (*inventorypb.ExternalExporter, error) {
+	var res *inventorypb.ExternalExporter
+	e := as.db.InTransaction(func(tx *reform.TX) error {
+		params := &models.CreateExternalExporterParams{
+			RunsOnNodeID: req.RunsOnNodeId,
+			ServiceID:    req.ServiceId,
+			Username:     req.Username,
+			Password:     req.Password,
+			Scheme:       req.Scheme,
+			MetricPath:   req.MetricPath,
```

Comment:
Those parameters are not required in API, and do not have default values. That's not okay.

I think we should default scheme to `http` and path to `/metrics` there, not rely on Prometheus defaults. We should also mention those defaults in API docs (.proto files).

And API tests should cover that.

Reply:
prometheus has default values for them, why shouldn't we rely on prometheus defaults?

Reply:
Because prometheus.base.yml can change default metrics path, and PMM inventory data will become invalid

Reply:
updated logic to set these values in model layer